src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -678,31 +677,30 @@ const getExampleLinks = (props: {
         }
       };
     }
+  } else if (section === SpeciesStatsSection.ASSEMBLY) {
+    // TODO: Change the type check to location when data type returned switches to location
+    const regionExample = exampleFocusObjects.find(
+      (object) => object.type === 'region'
```

Comment:
**Step 1:** unless I am mistaken, the only place that `region`s are currently coming into our app is from an api response that we are handling [here](https://github.com/Ensembl/ensembl-client/blob/85e253b136dba717d0acd1dd644287dff98c1307/src/shared/state/genome/genomeApiSlice.ts#L41). Let's change "region" to "location" in that piece of code.

**Step 2:** Line 683 above would then change to `object.type === 'location'`

**Step 3:** We probably won't have `region`s in `focusObjectSlice`.

Reply:
✅